src/features/Instructors/AddInstructors/index.jsx

```diff
@@ -99,12 +101,41 @@ const AddInstructors = ({ isOpen, onClose }) => {
     return null;
   };
 
+  const handleEditInstructor = async (e) => {
+    e.preventDefault();
+    try {
+      const formData = new FormData();
+      formData.append('enrollment_privilege', formState.instructor.hasEnrollmentPrivilege);
+      formData.append('institution_id', selectedInstitution.id);
+      formData.append('instructor_id', instructorInfo.instructorId);
+
+      await dispatch(editInstructor(formData));
+      handleCloseModal();
+      dispatch(updateInstructorAdditionRequest({ status: RequestStatus.INITIAL }));
+    } catch (error) {
+      logError(error);
+    }
+  };
+
   useEffect(() => {
     dispatch(updateInstructorAdditionRequest({ status: RequestStatus.INITIAL }));
 
     return () => dispatch(resetInstructorAdditionRequest());
   }, [dispatch]);
 
+  useEffect(() => {
+    if (isEditing && instructorInfo) {
```

Comment:
If I'm not mistaken `instructorInfo` always will be `true` because for the way in how this is being called on the colums file:

```javascript
// src/features/Instructors/InstructorsTable/columns.jsx
instructorInfo={{
    instructorName,
    instructorId,
    instructorEmail,
    hasEnrollmentPrivilege,
  }}
``` 

In the worst scenario we will get this: 

```javascript
{
  instructorName: undefined,
  instructorId: undefined,
  instructorEmail: undefined,
  hasEnrollmentPrivilege: undefined
}
``` 


In other words we have here a [Truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) value so if you want to check if we are editing **and** we **have** instructor info you should find another way to validate this.